use-shopping-cart/core/Entry.js

```diff
@@ -1,5 +1,6 @@
 import { isClient } from '../utilities/SSR'
-import { formatISO } from 'date-fns'
+// This is giving an error if I run any dev: script using pnpm
+// import { formatISO } from 'date-fns'
```

Comment:
Hi, @dayhaysoos Since I have commented import here so I can commit my code since pre commit hook is giving me an error. I need to uncomment it before merging this PR.  But if I uncomment this code and try to commit it, It is not working due to pre-commit hook which is giving me this error. 
![image](https://github.com/dayhaysoos/use-shopping-cart/assets/17196143/072f16c1-ac96-4b19-aeb1-da519f15afaa)